temporary_script.jl

```diff
@@ -0,0 +1,79 @@
+# this script should be removed when the alternative MO kernelmatrix is accepted or rejected. 
+
+using KernelFunctions, BenchmarkTools
+using LinearAlgebra
+
+mrank = 1
+dims = (in=5, out=3)
+x = [rand(dims.in) for _ in 1:20]
+
+xMOF = KernelFunctions.MOInputIsotopicByFeatures(x, dims.out)
+xMOO  = KernelFunctions.MOInputIsotopicByOutputs(x, dims.out)
+
+indk = IndependentMOKernel(GaussianKernel())
+
+Kind1 = kernelmatrix(indk, xMOF, xMOF)
+Kind2 = kernelmatrix2(indk, xMOF, xMOF)
+
+Kind1 ≈ Kind2
```

Comment:
How about turning this into an actual test using `invoke` (after renaming `kernelmatrix2` back to `kernelmatrix`)?
e.g.
```suggestion
K = kernelmatrix(indk, xMOF, xMOF)
K_fallback = invoke(kernelmatrix, Tuple{Kernel, AbstractVector, AbstractVector})(indk, xMOF, xMOF)

@test K ≈ K_fallback
```

Reply:
I take it back, this should already be covered by
```julia
    @test kernelmatrix(k, x0, x2) ≈ [k(xl, xr) for xl in x0, xr in x2]
```
in src/test_utils.jl